### PR TITLE
fix(acme): fix templating and add unit tests (#655)

### PR DESCRIPTION
**Pull request by linoleparquet**
Related to #655 

Hi folks !
Fisrt PR on open source project. 
I renamed `acme_test_secret` to `acme-test-secret` because helm lint raised a warning.

### CODE REVIEW

File: charts/acme/tests/defaults_test.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+suite: default tests
+values:
+  - ../values.yaml
+  - secrets_default_values.yaml
```
Comment (dduportal) [inline charts/acme/tests/defaults_test.yaml:4]:
```suggestion
```

The "default test" should not provide any (helm) value at all as it is expected to simulate the `helm template ./` command.

I believe it *might* break the test and could require more changes (sorry 😅)

**Review by dduportal — CHANGES_REQUESTED** [review]
Thanks for the proposed contribution!

It is in a good direction but require a bit more work (see my suggestion and comment) as we expect the default unit test suite to not provide any values

File: charts/acme/tests/defaults_test.yaml (hunk 1 above)
Comment (linoleparquet) [reply charts/acme/tests/defaults_test.yaml]: Got your point
I split the unit test in two files: custom test and default test. Let me know if that looks good to you.

I actually took example of the accountapp chart to redact theses tests, which doesn't follow this structure. If you create an issue I can fix it !

**Review by dduportal — APPROVED** [review]
LGTM, many thanks!

**Pull request merged**